prisma/migrations/01_functions/migration.sql

```diff
@@ -33,7 +30,7 @@ SET
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */
 ;
 
-CREATE DEFINER = `root` @`%` PROCEDURE `add_donation`(
+CREATE PROCEDURE `add_donation`(
```

Comment:
Need to be root user defining the object to be able specify root as the definer, which makes sense.

Omitting the explicit definer will make the current user the definer when running this. It will technically be a difference between the databases, but a good and consistent one I think. And this schema definition stays the same.